README.md

```diff
@@ -104,6 +104,52 @@ systemctl status MTProxy.service
 systemctl enable MTProxy.service
 ```
 
+### Timers
+
+This provides automatic proxy-multi.conf file update everyday
+
+1. Create and edit systemd service file (especially path of proxy-multi.conf file):
+```bash
+nano /etc/systemd/system/MTProxy-multiUpdater.service
+```
+
+```bash
+[Unit]
+Description=Update MTPRoxy proxy-multi.conf file
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/curl -s https://core.telegram.org/getProxyConfig -o /opt/MTProxy/proxy-multi.conf
+```
+2. Add the timer (this example runs above service everyday at 4AM)
+```bash
+nano /etc/systemd/system/MTProxy-multiUpdater.service
```

Comment:
It's possible that the correct file name should be `MTProxy-multiUpdater.timer`

Reply:
Done in commit 9b018dcefb01494fb8a306a2fc2d1302c10f25d3